addons/test_mail/tests/test_message_post.py

```diff
@@ -264,6 +254,18 @@ def test_notify_recipients_internals(self):
                 self.assertFalse(partner_info['has_button_access'])
                 self.assertFalse(emp_info['has_button_access'])
 
+                # test on falsy records (False model cannot be browsed, skipped)
+                if model:
+                    record_falsy = self.env[model].browse(res_id)
+                    classify_res = record_falsy._notify_get_recipients_classify(
+                        pdata, 'Test', msg_vals=notify_msg_vals)
+                    # find back information for partner
+                    partner_info = next(item for item in classify_res if item['recipients'] == self.partner_1.ids)
+                    emp_info = next(item for item in classify_res if item['recipients'] == self.partner_employee.ids)
+                    # check there is no access button
+                    self.assertFalse(partner_info['has_button_access'])
+                    self.assertFalse(emp_info['has_button_access'])
```

Comment:
Seems duplicated, but maybe this test will be modified again in a later commit to introduce some changes based on model / no model.